flink-connector-dynamodb/pom.xml

```diff
@@ -132,5 +132,12 @@ under the License.
             <scope>test</scope>
         </dependency>
 
+        <dependency>
+            <groupId>org.apache.flink</groupId>
+            <artifactId>flink-table-planner-loader</artifactId>
```

Comment:
previously we had separate modules for these IT tests, they were more of E2E tests for uber sql connector. since we are not doing this with this approach are we intending to have another E2E test?

Reply:
Yes, I was thinking that E2E test will be testing against an actual DynamoDB whereas this IT case is testing against a local DynamoDB container.

Reply:
I am happy to follow this approach if we intend to have E2E tests, they need not to be against an actual DDB (this is preferable ofc) but we need to verify that the connector works when packaging the uber jar to sql client.

KDS tests for reference
https://github.com/apache/flink-connector-aws/blob/2a24e7c1e8d2a8f613838fd0419177d4f96991bf/flink-connector-aws-e2e-tests/flink-connector-aws-kinesis-streams-e2e-tests/pom.xml#L99